pages/education.py

```diff
@@ -84,64 +120,127 @@ def render_education_content(tab):
                         )
                     )
                 ),
+                dbc.Row(
+                    [
+                        # How has total expenditure changed over time?
+                        dbc.Col(
+                            [
+                                html.P(
+                                    id="education-narrative",
+                                    children="loading...",
+                                    style={"height": "100px", "margin": "10px"},
+                                ),
+                                dcc.Graph(
+                                    id="education-total",
+                                    config={"displayModeBar": False},
+                                ),
+                            ],
+                            xs={"size": 12, "offset": 0},
+                            sm={"size": 12, "offset": 0},
+                            md={"size": 12, "offset": 0},
+                            lg={"size": 6, "offset": 0},
+                            style={"marginBottom": "3rem"},
+                        ),
+                        dbc.Col(
+                            [
+                                html.P(
+                                    id="education-outcome-narrative",
+                                    children="loading...",
+                                    style={"height": "100px", "margin": "10px"},
+                                ),
+                                dcc.Graph(
+                                    id="education-outcome",
+                                    config={"displayModeBar": False},
+                                ),
+                            ],
+                            xs={"size": 12, "offset": 0},
+                            sm={"size": 12, "offset": 0},
+                            md={"size": 12, "offset": 0},
+                            lg={"size": 6, "offset": 0},
+                        ),
+                    ],
+                ),
+                dbc.Row(
+                    dbc.Col(
+                        html.Hr(),
+                    )
+                ),
                 dbc.Row(
                     dbc.Col(
-                        html.P(
-                            id="education-narrative",
-                            children="loading...",
+                        html.H3(
+                            children="Public Spending Breakdown by Education Sector",
```

Comment:
I might be mistaken, but are we including this figure finally? I have a slight recollection that they didnt want this breakdown treemap. Just checking.

Reply:
It depended on if the narrative fits in this context. 
I personally liked the idea of having larger picture above and going into the breakdown below on the sub func cateogories. But let's take a vote!  @weilu @bhupatiraju 

![image](https://github.com/user-attachments/assets/28ee56e8-a7fc-4123-a0c4-d0ce32f538b4)

Reply:
Hmm.. 
It seems like in either case, we need to set the height for the narrative for the alignment..
I kept the same layout, since it will be more flexible to handle narrative/figure as one set (as opposed to the suggested grid structure).
<img width="1332" alt="image" src="https://github.com/user-attachments/assets/3619d3d8-2710-4392-b5c0-884b5a127813">

Reply:
Decided to keep this for now after a chat with @weilu